### PR TITLE
Add all function

### PR DESCRIPTION
**Pull request by dhev-wisdom**
Added a 'all' function using built in all function in the ivy package
Close https://github.com/unifyai/ivy/issues/16708

### CODE REVIEW

File: ivy/functional/frontends/paddle/tensor/tensor.py [hunk 1]
```diff
@@ -155,3 +155,7 @@ def floor(self, name=None):
     )
     def tanh(self, name=None):
         return ivy.tanh(self._ivy_array)
+
+    @with_unsupported_dtypes({"2.4.2 and below": ("float16", "bfloat16")}, "paddle")
+    def all(self, name=None):
```
Comment (Sam-Armstrong) [inline ivy/functional/frontends/paddle/tensor/tensor.py:160]: This should have the kwargs axis=None, keepdim=False, name=None (from paddle docs https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/Tensor_en.html)

**Review by Sam-Armstrong — CHANGES_REQUESTED** [review]
Hey @dhev-wisdom, please could you link your issue to https://github.com/unifyai/ivy/issues/15115 by leaving a comment in the format  "- [ ] #issue_number" on https://github.com/unifyai/ivy/issues/15115? 

Also please could you write a test for this function? You can find out how to do this here https://unify.ai/docs/ivy/overview/deep_dive/ivy_frontends_tests.html#ivy-frontend-tests.

**Comment by Sam-Armstrong** [comment]
Close https://github.com/unifyai/ivy/issues/16708

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by Sam-Armstrong** [comment]
closed stale pull request

**Pull request closed without merging**